File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

/**
 * 品牌服务层接口
 */
public interface BrandService {
    /**
     * 查询所有品牌
     * @return
     */
//    List<TbBrand> findAll();
    PageResult findPage(TbBrand tbBrand,int page,int size);
    void add(TbBrand tbBrand);
    void update(TbBrand tbBrand);
    void delete(long[] ids);

    List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {
    this.save=function (entity) {
        return $http.post('../brand/save.do',entity)
    }
    this.queryPage=function (page, rows, entity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,entity);
    }
    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }
})